libd-llvm/src/d/llvm/backend.d

```diff
@@ -119,22 +117,15 @@ public:
 		auto targetData = LLVMGetTargetMachineData(targetMachine);
 		LLVMAddTargetData(targetData, pm);
 		LLVMPassManagerBuilderPopulateModulePassManager(pmb, pm);
-		LLVMRunPassManager(pm, dmodule);
-		
-		// Dump module for debug purpose.
-		// LLVMDumpModule(dmodule);
```

Comment:
forgot about this. 
(I deleted it, as I thought one could now easily do this with --emit-llvm without rebuilding sdc, but sure)